### ISSUE TITLE
[Bug]: Delayed activation of Remind me Later Key in Obsidian 2.0

### ISSUE THREAD

**Issue by aubreyz** [uphy/obsidian-reminder#109]
### Describe the bug

Since Obsidian 2.0 it is taking about 10 seconds after the dialog displays for the Remind Me Later button to become active. For teh first 10 seconds it is not pressable

![image](https://user-images.githubusercontent.com/25100168/196394839-86ae38bb-2f3a-447b-80f0-205673d82a7f.png)
 

### Expected Behavior

It should be immediate as before

### Steps to reproduce

Upgrade to Obsidian 2.0

### Operating system

Windows

**Comment by MSBack** [comment]
@aubreyz Obsidian is on version 1.3.4, what are you talking about with Obsidian 2.0?

**Comment by aubreyz** [comment]
1.0 which was released ion Oct 2022 when this bug report was made. No problem anyway I have stopped using due to the unresolved bugs (elimination of tags, deletion of dates, and severe conflicts with the task plugin). A great plugin idea and unique but sadly cannot use.

**Comment by MSBack** [comment]
> 1.0 which was released ion Oct 2022 when this bug report was made. No problem anyway I have stopped using due to the unresolved bugs (elimination of tags, deletion of dates, and severe conflicts with the task plugin). A great plugin idea and unique but sadly cannot use.

Yeah, it seems a bit unsafe. Navigating the issues has me wondering if I also should stop using it.

Anyway, yes 1.0, but you wrote 2.0 above.

**Comment by aubreyz** [comment]
> Anyway, yes 1.0, but you wrote 2.0 above.

A Typo :). 

I would really love it if the developers of this plugin could address the more glaring issues because it is a great plugin otherwise and there is no other like it.
